File: Examples/Mvc.CS/Models/OverviewViewModel.cs
using System.Collections.Generic;
using System.Web.Mvc;
using GleamTech.ExamplesCore;

namespace GleamTech.DocumentUltimateExamples.Mvc.CS.Models
{
    public class OverviewViewModel
    {
        public OverviewViewModel()
        {
            OutputFormats = new Dictionary<string, List<SelectListItem>>();
        }

        public ExampleFileSelector ExampleFileSelector { get; set; }

        public string InputFormat { get; set; }

        public string ConvertHandlerUrl { get; set; }

        public Dictionary<string, List<SelectListItem>> OutputFormats { get; set; }
    }
}

File: Examples/Mvc.VB/Models/OverviewViewModel.vb
Imports System.Collections.Generic
Imports GleamTech.ExamplesCore

Namespace Models

    Public Class OverviewViewModel
        Public Sub New()
            OutputFormats = New Dictionary(Of String, List(Of SelectListItem))()
        End Sub

        Public Property ExampleFileSelector As ExampleFileSelector

        Public Property InputFormat As String

        Public Property ConvertHandlerUrl As String

        Public Property OutputFormats As Dictionary(Of String, List(Of SelectListItem))
    End Class
End NameSpace